ivlsrc/SimDTM.cc

```diff
@@ -0,0 +1,113 @@
+// See LICENSE.SiFive for license details.
```

Comment:
I see you fixed the duplication of VSRC files, but same goes for CSRC...you should not need to add copies of these files. If there are minor differences required you can use `#ifdef IVLSIM` or such to handle those.